newIDE/app/src/MainFrame/index.js

```diff
@@ -304,6 +312,22 @@ class MainFrame extends React.Component<Props, State> {
 
   openFromPathOrURL = (url: string, cb: Function) => {
     const { i18n } = this.props;
+
+    const autoSavePath = url + '.autosave';
+    if (fs.existsSync(autoSavePath)) {
+      const autoSavedTime = fs.statSync(autoSavePath).mtime.getTime();
+      const saveTime = fs.statSync(url).mtime.getTime();
+      if (autoSavedTime > saveTime) {
+        //eslint-disable-next-line
+        const answer = confirm(
+          i18n._(
+            `Autosave newer than the project file exists. Would you like to load it?`
+          )
+        );
+        if (answer) url += '.autosave';
```

Comment:
`if (answer) url = autoSavePath;`

This is slightly clearer and avoid repeating the ".autosave" (will be useful if we want to change the name later)

Reply:
In that case I still need to keep the original url, so as not to set the project path to the autosave file later on
`this.state.currentProject.setProjectFile(url);`